### PR TITLE
Omit UI documentation

### PR DESCRIPTION
**Pull request by jsong785**

### CODE REVIEW

**Review by paul-england — APPROVED** [review]

File: docs/references/ayx-plugin-cli.md [hunk 1]
```diff
@@ -215,14 +215,25 @@ The plugin version.
 -   Type: String
 -   Permissible Values: Any
 
+#### \--omit-ui
+
+Do not generate UI artifacts for this plugin. UI artifacts can be generated later by using `generate-ui` command.
```
Comment (ayx-david) [inline docs/references/ayx-plugin-cli.md]: TW review:

Don't generate UI artifacts for this plugin. You can generate UI artifacts later with the `generate-ui` command.

File: docs/references/ayx-plugin-cli.md [hunk 2]
```diff
@@ -215,14 +215,25 @@ The plugin version.
...
 ### Parameters
 
-This command takes no parameters.
+#### \--omit-ui
+
+Do not build any UI artifacts for the plugins in the workspace.
+
+- If a plugin does not have any UI artifacts, nothing will happen.
+- If a plugin has UI artifacts but it was never built, nothing will happen.
+- If a plugin has UI artifacts that was previously built, the previous build result will be packaged.
```
Comment (ayx-david) [inline docs/references/ayx-plugin-cli.md]: TW review:

Don't build any UI artifacts for the plugins in the workspace.

- If a plugin doesn't have any UI artifacts, nothing happens.
- If a plugin has UI artifacts, but it was never built, nothing happens.
- If a plugin has UI artifacts and was previously built, the previous build result is packaged.

File: docs/references/ayx-plugin-cli.md [hunk 3]
```diff
@@ -278,6 +289,10 @@ Note that admin installation requires admin access.
 -   Permissible Values: user, admin
 -   Default: user
 
+#### \--omit-ui
+
+see [`--omit-ui` for --create-yxi](#create-yxi)
```
Comment (ayx-david) [inline docs/references/ayx-plugin-cli.md]: TW review:

Go to [`--omit-ui` for --create-yxi](#create-yxi).

File: docs/references/ayx-plugin-cli.md [hunk 4]
```diff
@@ -417,6 +443,9 @@ to define these items:
     versions of a tool so that users can choose to upgrade to a new
     version or use the previous one inside of any Alteryx workflows.
 
+:information_source:
+Use `--omit-ui` argument if this plugin will not have UI. UI can be generated later by using the [generate-ui](#generate-ui) command.
```
Comment (ayx-david) [inline docs/references/ayx-plugin-cli.md]: TW review:

Use the `--omit-ui` argument if this plugin doesn't have a UI. You can generate the UI later with the [generate-ui](#generate-ui) command.

File: docs/references/faq.md [hunk 5]
```diff
@@ -279,3 +279,19 @@ error\...
 2.  Next, go to the \\backend sub-folder and add this line to the
     **requirements-thirdparty.txt** file: `protobuf==3.20.1`.
 3.  Rebuild the plugin.
+
+## _"My plugin doesn't need UI"_
+
+Just simply delete the UI folder. For new plugins, [use --omit-ui](ayx-plugin-cli.md#create-ayx-plugin)
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

If your plugin doesn't need a UI, you can simply delete the UI folder. For new plugins, [use --omit-ui](ayx-plugin-cli.md#create-ayx-plugin).

File: docs/references/faq.md [hunk 6]
```diff
@@ -279,3 +279,19 @@ error\...
...
+
+## _"I built my plugin without UI, but change my mind"_
+
+You can [create a UI for a given workspace or specific plugin](ayx-plugin-cli#generate-ui)
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review: add a period.

File: docs/references/faq.md [hunk 7]
```diff
@@ -279,3 +279,19 @@ error\...
 2.  Next, go to the \\backend sub-folder and add this line to the
     **requirements-thirdparty.txt** file: `protobuf==3.20.1`.
 3.  Rebuild the plugin.
+
+## _"My plugin doesn't need UI"_
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

## _"My plugin doesn't need a UI."_

File: docs/references/faq.md [hunk 8]
```diff
@@ -279,3 +279,19 @@ error\...
...
+
+Just simply delete the UI folder. For new plugins, [use --omit-ui](ayx-plugin-cli.md#create-ayx-plugin)
+
+## _"I built my plugin without UI, but change my mind"_
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

## _"I built my plugin without a UI but changed my mind."_

File: docs/references/faq.md [hunk 9]
```diff
@@ -279,3 +279,19 @@ error\...
...
+
+You can [create a UI for a given workspace or specific plugin](ayx-plugin-cli#generate-ui)
+
+## _"I don't want to keep rebuilding UI components when creating a yxi"_
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

## _"I don't want to rebuild UI components when creating a YXI."_

File: docs/references/faq.md [hunk 10]
```diff
@@ -279,3 +279,19 @@ error\...
...
+
+You can do this by using [-omit-u in create-yxi](ayx-plugin-cli#create-yxi)
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

To address this, use [-omit-u in create-yxi](ayx-plugin-cli#create-yxi).

File: docs/references/faq.md [hunk 11]
```diff
@@ -279,3 +279,19 @@ error\...
...
+
+## _"I don't want to keep rebuilding UI components installing a yxi through the CLI"_
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

## _"I don't want to rebuild UI components when installing a YXI through the CLI."_

File: docs/references/faq.md [hunk 12]
```diff
@@ -279,3 +279,19 @@ error\...
...
+
+You can do this by using [-omit-u in designer-install](ayx-plugin-cli#designer-install)
```
Comment (ayx-david) [inline docs/references/faq.md]: TW review:

To address this, use [-omit-u in designer-install](ayx-plugin-cli#designer-install).

**Review by ayx-david-ung — APPROVED** [review]
LGTM

**Pull request merged**